test-case/src/main/java/pkg/Issue68.java

```diff
@@ -0,0 +1,14 @@
+package pkg;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** @see https://github.com/KengoTODA/findbugs-slf4j/issues/68 */
+public class Issue68 {
+  private final Logger log = LoggerFactory.getLogger(getClass());
+
+  void method() {
+    log.info("I have a placeholder but no parameter. {}");
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Not enough arguments. ([squid:S2275](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS2275))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.findbugs.slf4j%3Afindbugs-slf4j&pullRequest=143&issues=AWZdIAKprFHbA8-ERVUx&open=AWZdIAKprFHbA8-ERVUx)

[ISSUE_KEY]: AWZdIAKprFHbA8-ERVUx